mmv1/products/edgecontainer/Cluster.yaml

```diff
@@ -0,0 +1,515 @@
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 60
+      update_minutes: 60
+      delete_minutes: 60
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The location of the resource.
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The GDCE cluster name.
+import_format: ["projects/{{project}}/locations/{{location}}/clusters/{{name}}"]
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "edgecontainer_cluster"
+    primary_resource_id: "default"
+    vars:
+      edgecontainer_cluster_name: "basic-cluster"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "edgecontainer_cluster_with_maintenance_window"
+    primary_resource_id: "default"
+    vars:
+      edgecontainer_cluster_name: "cluster-with-maintenance"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "edgecontainer_local_control_plane_cluster"
+    primary_resource_id: "default"
+    skip_test: true
+    vars:
+      edgecontainer_cluster_name: "local-control-plane-cluster"
+properties:
+  - !ruby/object:Api::Type::Time
+    name: "createTime"
+    description: |
+      The time the cluster was created, in RFC3339 text format.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: "updateTime"
+    description: |
+      The time the cluster was last updated, in RFC3339 text format.
+    output: true
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: "labels"
+    default_from_api: true
+    description: "User-defined labels for the edgecloud cluster."
+  - !ruby/object:Api::Type::NestedObject
+    name: "fleet"
+    required: true
+    immutable: true
+    description: |
+      Fleet related configuration.
+      Fleets are a Google Cloud concept for logically organizing clusters,
+      letting you use and manage multi-cluster capabilities and apply
+      consistent policies across your systems.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "project"
+        immutable: true
+        description: |
+          The name of the Fleet host project where this cluster will be registered.
+          Project names are formatted as
+          `projects/<project-number>`.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: "membership"
+        default_from_api: true
+        immutable: true
```

Comment:
I think this can just be changed to `output: true`, based on the API docs. That way a user can't try to set it.

Reply:
Thanks for capturing, added output: true and remove the immutable